test/arrays/ferguson/semantic-examples/valgrind-futures/4-return-alias-local.chpl

```diff
@@ -1,7 +1,7 @@
 use samples;
 proc f() {
   var A:[1..4] R;
-  var B => A;
+  ref B = A;
```

Comment:
Unless you have some particular reason to keep this around, I'd prefer you simply remove this test. The new version no longer matches the intent of the test, since the array will now return by value. I think it'd be better covered by any future along the lines of issue #5341.